client/src/shared/components/Iframe/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+interface IframeProps {
+  src: string;
+  className?: string;
+  width?: string;
+  height?: string;
+  title?: string;
+}
+
+const Iframe: React.FunctionComponent<IframeProps> = ({
+  src,
+  className,
+  width,
+  height,
+  title
+}: IframeProps) => {
+  return (
+    <div>
+      <iframe
+        className={className}
+        width={width}
+        height={height}
+        src={src}
+        title={title}
+      ></iframe>
+    </div>
+  );
+};
+
+Iframe.defaultProps = {
+  className: 'mx-auto m-2',
```

Comment:
- move this to the classname attrbute. And prop should empty 
```suggestion
      <iframe
        className={`mx-auto m-2 ${className}`}
        width={width}
        height={height}
        src={src}
        title={title}
      ></iframe>
    </div>
  );
};

Iframe.defaultProps = {
  className: '',
```